axelrod/tests/strategies/test_player.py

```diff
@@ -5,12 +5,13 @@
 import itertools
 import numpy as np
 
-import axelrod
-from axelrod import DefaultGame, MockPlayer, Player, simulate_play
+from axelrod import DefaultGame, MockPlayer, Player, simulate_play, Match, seed as axlseed
```

Comment:
Can you just import axelrod and use verbose calls to the specific elements. For example:

```
import axelrod as axl
axl.seed(0)
axl.MockPlayer
axl.Cooperator()
```

This helps keep things consistent.